test_runner/regress/test_pageserver_secondary.py

```diff
@@ -555,6 +556,7 @@ def test_secondary_downloads(neon_env_builder: NeonEnvBuilder):
     )
 
 
+@pytest.mark.skipif(os.environ.get("BUILD_TYPE") == "debug", reason="only run with release build")
```

Comment:
Why do we skip this in debug mode? It doesn't seem to be doing anything that intensive.

Reply:
It's two things:
- It does enough writes to generate some layers, and sometimes debug builds are _really_ slow, to the point that we regret trying to write a few megs of layers
- The final stage of the test assumes that with the failpoint removed the download will complete within a set wait_ms, which is hard to reason about in debug builds